security-privacy-self-assessment.md

```diff
@@ -0,0 +1,77 @@
+# Security and Privacy Questionnaire
+
+[Security and Privacy questionnaire](https://www.w3.org/TR/security-privacy-questionnaire/)
+responses for the Device Posture API
+
+### 2.1. What information might this feature expose to Web sites or other parties, and for what purposes is that exposure necessary?
+
+This API exposes the posture of the device, a physical state of a device. The posture typically apply only to device with a foldable screen or dual screens devices (devices with two screens connected with a physical hinge). The posture is typically derived from the hinge angle sensor but also few other sensors (for example if the keyboard is detached or not).

```

Comment:
but can include data from other sensors as well?

Reply:
What do you mean?

Reply:
On Android for example https://www.w3.org/TR/device-posture/#foldables isn't true if the browser window is running on an external screen (the OS tells us continuous), despite the device being physically folded. Also on Android if you do a side by side app window (like say the browser and another app) the OS will tell us "continuous" because really there isn't much value in trying to layout stuff in half the screen.

Reply:
you say, it is typically derived from the hinge angle sense, but then you also say a few other sensors... so is it always derived from all of these? 

The posture is typically derived from the hinge angle sensor, but additional data can be used to determine the posture such as whether the keyboard is attached or not. 

also few other sensors (for example if the keyboard is detached or not).

Reply:
On Android it's hinge angle + window state. On Windows, it's Hinge Angle + other sensors.

Reply:
But I don't think we should enter the specifics of each OS so let's keep generic. I think actually saying the posture is computed using a set of sensors sounds better.